frontends/ol-components/src/components/ThemeProvider/chips.tsx

```diff
@@ -76,6 +76,9 @@ const chipComponent: NonNullable<ThemeOptions["components"]>["MuiChip"] = {
       style: {
         borderColor: colors.silverGrayLight,
         color: colors.darkGray1,
+        "&.Mui-focusVisible": {
+          backgroundColor: "transparent",
+        },
```

Comment:
This and other changes here avoid mui's default focus-visible style, which was problematic for one of our variants.

![Uploading Screenshot 2024-06-10 at 11.35.44 AM.png…]()